server/controllers/items/recent_public.coffee

```diff
@@ -0,0 +1,40 @@
+__ = require('config').universalPath
+_ = __.require 'builders', 'utils'
+error_ = __.require 'lib', 'error/error'
+items_ = __.require 'controllers', 'items/lib/items'
+user_ = __.require 'controllers', 'user/lib/user'
+
+module.exports = (req, res)->
+  { query } = req
+  assertImage = query['assert-image'] is 'true'
+  reqUserId = req.user?._id
+
+  items_.publicByDate 100, 0, assertImage, reqUserId
+  .then selectRecentItems
+  .then bundleOwnersData.bind(null, res, reqUserId)
+  .catch error_.Handler(req, res)
+
+selectRecentItems = (items)->
+  itemsLimit = 15
+  maxItemsPerOwner = 3
```

Comment:
this kind of hardcoded parameters could be defined just below the `require`s

Reply:
or actually, they would probably be better defined dynamically from the query string (like how the `limit` is defined for the `last-public` action), so that the client can tell how many items it want, depending on the screen width